src/main/java/com/couchbase/client/spring/cache/CouchbaseCacheManager.java

```diff
@@ -101,6 +103,39 @@ public void setDefaultCacheBuilder(CacheBuilder defaultCacheBuilder) {
     this.defaultCacheBuilder = defaultCacheBuilder;
   }
 
+  /**
+   * Register an additional cache with the specified name using the default builder.
+   * <p>
+   * Caches can only be configured at initialization time. Once the cache manager
+   * has been initialized, no cache can be further prepared.
+   * @param name the name of the cache to add
+   * @throws IllegalStateException if no builder is available or if the cache manager
```

Comment:
I don't know if you use that dynamic term elsewhere but feel free to update the doc any way you like.